### PR TITLE
Add column support for Raptor

### PR DESCRIPTION
**Pull request by joy-yao**

### CODE REVIEW

**Comment by joy-yao** [comment]
Thanks @electrum for your review! I've made the changes in the last commit. Please review again.

File: presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardManager.java [hunk 1]
```diff
@@ -32,6 +32,11 @@
     void createTable(long tableId, List<ColumnInfo> columns);
 
     /**
+     * Add a column to the end of the table
```
Comment (electrum) [inline presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardManager.java]: Nit: end Javadoc in period

**Comment by electrum** [comment]
Looks good. Please squash last commit before pushing

**Comment by electrum** [comment]
The Travis build failed, though seems unrelated. Please make sure a command line build for presto-raptor works.

**Comment by joy-yao** [comment]
Thanks @electrum , local build and test passed. Merging.

**Pull request merged**